CHANGELOG.rst

```diff
@@ -1,6 +1,9 @@
 3.0.7 (unreleased)
 ==================
 
+* Fix issue when trying to import the `anydbm` module. (`#2248`_).
```

Comment:
Probably this can be made more general, since it is not exclusive to `anydbm`. Perhaps: *Fix issue in assertion rewriting breaking due to modules silently discarding other modules when importing fails*?